.github/workflows/push.yaml

```diff
@@ -0,0 +1,373 @@
+          else
+            REV_RANGE=${PREVIOUS_TAG}..${LATEST_TAG}
+          fi
+          git log --pretty=format:"- %s %H (%aN)" --no-merges ${REV_RANGE} > ${VERSION}-CHANGELOG.txt
+          cat ${VERSION}-CHANGELOG.txt
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        with:
+          body_path: ${{ env.VERSION }}-CHANGELOG.txt
+          draft: false
+          prerelease: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  release-operatorhub:
+    name: Relase to OperatorHub via community-operators (We don't actually commit in this case)
+    runs-on: ubuntu-latest
+    if: "startsWith(github.ref, 'refs/tags')"
+    needs: ["build"]
+    steps:
+
+      - name: set repo name
+        shell: bash
+        run: | 
+          echo "REPOSITORY_NAME=$(basename $GITHUB_REPOSITORY)" >> $GITHUB_ENV
+
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.14
+      - name: Download operator sdk
+        shell: bash
+        env:
+          RELEASE_VERSION: v1.1.0
+        run: | 
+          curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+          chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+          mkdir ${HOME}/bin
+          mv operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu ${HOME}/bin/operator-sdk
+          echo "${HOME}/bin" >> $GITHUB_PATH 
+      - name: Get the version
+        id: get_version
+        run: |
+          export TAG=${GITHUB_REF/refs\/tags\//}
+          echo "VERSION=${TAG:1}" >> $GITHUB_ENV
+      - name: checkout community-operators
+        uses: actions/checkout@v2
+        with:
+          repository: operator-framework/community-operators
+          path: ./tmp/community-operators
+
+      - name: check whether it is first release
+        shell: bash
+        run: |
+          echo first_release=$([[ ! -d "./tmp/community-operators/community-operators/$(basename $GITHUB_REPOSITORY)" ]] && echo 'true' || echo 'false') >> $GITHUB_ENV
+          echo $first_release
+
+      
+      - name: create and copy bundle to community operators
+        shell: bash
+        run: | 
+          make bundle IMG=quay.io/${{ github.repository_owner }}/$(basename $GITHUB_REPOSITORY):v${VERSION} VERSION=${VERSION} DEFAULT_CHANNEL=alpha
+          sed  's/bundle\///g' bundle.Dockerfile > bundle/Dockerfile
+          sed -i '/replaces: '"$(basename $GITHUB_REPOSITORY)"'/d' ./bundle/manifests/$(basename $GITHUB_REPOSITORY).clusterserviceversion.yaml
+          mkdir -p ./tmp/community-operators/community-operators/$(basename $GITHUB_REPOSITORY)/${VERSION}
+          /bin/cp -v -R ./bundle/* ./tmp/community-operators/community-operators/$(basename $GITHUB_REPOSITORY)/${VERSION}         
+
+      # - name: Create Pull Request
```

Comment:
Can this section be removed?

Reply:
I don't understand you want to remove line 242? for what reason?

Reply:
no. starting at 245. the entire commented out section

Reply:
fixed. It needed to be un-commeted.